### PR TITLE
Feature#28/reports

### LINKED ISSUE #28: Reports
# REQUIREMENTS
A report view of people with incidents and another of places with incidents will be created

The data required for people:
institution
department
username
symptomatic
Covid +
Covid -

The data required for places:
Institution
area
place
symptomatic
Covid +
Covid -

# DESIGN
imagen

# Acceptance Criteria
List all required acceptance criteria
- [ ] For all reports show a date range that can be specified manually.
- [ ] Manual filtering by institution, area, department or place as the case may be.
- [ ] By default show the summary of the last ten days.
- [ ] Sorting from highest to lowest incidence.
- [ ] The highest incidence will be for covid positive cases, second for symptomatic ones.
- [ ] Dynamic search
- [ ] Export reports

### PR DESCRIPTION
**Pull request by mariormt17**
# Description
In this PR, we create the requested reports.
- The first type of report is about people with incidents.
- The second type of report is of places with incidents.
  
  ## Screenshots 
imagen
imagen

### CODE REVIEW

**Review by FranckZnt19 — APPROVED** [review]
Nice code, very clean.

**Review by ramasoftdev — APPROVED** [review]
LGTM, missing yet the tests; but is OK

File: alerta-covid/app/controllers/reports_controller.rb [hunk 1]
```diff
@@ -18,4 +19,20 @@ def get_reports_by_users(commit)
...
+    when "2"
+      Place.all.order(:area_id)
+    when "1"
+      Place.all.order(:id)
+    else
+      places = Place.all
+      places.sort_by{ |place|
+        place.incidences.select{ |incidence| incidence.result == true }.count
+      }.reverse
+    end
```
Comment (ArielMD) [inline alerta-covid/app/controllers/reports_controller.rb:36]: Nice filter method

**Review by ArielMD — APPROVED** [review]
Good work team, these reports are excellent. now we just need to add styles

**Pull request merged**